src/envoy/http/authn/peer_authenticator.cc

```diff
@@ -40,6 +40,15 @@ bool PeerAuthenticator::run(Payload* payload) {
   for (const auto& method : policy_.peers()) {
     switch (method.params_case()) {
       case iaapi::PeerAuthenticationMethod::ParamsCase::kMtls:
+        if (method.mtls().mode() == iaapi::MutualTls_Mode_PERMISSIVE) {
```

Comment:
This is logged at runtime, correct? Isn't it going to spam the logs?

Reply:
Yes, it's logged at runtime, I think moving it to log when the authN filter is initialized should be better, though it might be ignored very easily as there will only be 1 line of it.

It's best to do this in the validation stage but the last time I checked it doesn't support returning a warning message, but only an error message that would prevent the yaml to be applied.